src/components/App/App.tsx

```diff
@@ -128,6 +133,7 @@ class App extends React.Component<Props> {
                         toggleFavorite={this.toggleFavorite} 
                         saveNote={this.saveNote} 
                         favorites={this.state.favorites} 
+                        resetHome={() => this.setState({ currentList: this.state.apiList })}
```

Comment:
Definitely could be wrong, but I think currentApis is the state that we've gotta reset, cant find currentList being used anywhere else